SURVEY.md

```diff
@@ -6,7 +6,7 @@ We need to figure out pagination, order, pacing, and reasonable length, in addit
 ## How do you help build the internet?
 
 ### Position
-    What role do you play? (wording)
+    What role(s) do you play? (wording)
```

Comment:
@paulsheldrake if you can't think of any better work either, can you just remove the '(wording)' and we'll call it good?  Thx